init.ts

```diff
@@ -503,7 +512,7 @@ try {
 
 let FRESH_CONFIG_TS = `import { defineConfig } from "$fresh/server.ts";\n`;
 if (useTwind) {
-  FRESH_CONFIG_TS += `import twindPlugin from "$fresh/plugins/twind.ts"
+  FRESH_CONFIG_TS += `import twindPlugin from "$fresh/plugins/twindv1.ts"
```

Comment:
Thanks for this. This patch is missing in https://fresh.deno.dev/docs/examples/using-twind-v1 docs.